driver-core/src/test/functional/com/mongodb/internal/connection/TestCommandListener.java

```diff
@@ -36,17 +36,21 @@
 import org.bson.codecs.configuration.CodecRegistry;
 
 import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.List;
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.locks.Condition;
 import java.util.concurrent.locks.Lock;
 import java.util.concurrent.locks.ReentrantLock;
 
+import static java.util.Collections.emptyList;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNull;
 import static org.junit.Assert.assertTrue;
 
 public class TestCommandListener implements CommandListener {
+    private final List<String> eventTypes;
```

Comment:
See https://github.com/mongodb/specifications/blob/master/source/unified-test-format/unified-test-format.rst#entity-client for the necessity of these new properties.